zipkin-storage/elasticsearch/src/main/java/zipkin2/elasticsearch/ElasticsearchSpecificTemplates.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright The OpenZipkin Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package zipkin2.elasticsearch;
+
+import static zipkin2.elasticsearch.ElasticsearchVersion.V5_0;
+import static zipkin2.elasticsearch.ElasticsearchVersion.V6_0;
+import static zipkin2.elasticsearch.ElasticsearchVersion.V6_7;
+import static zipkin2.elasticsearch.ElasticsearchVersion.V7_0;
+import static zipkin2.elasticsearch.ElasticsearchVersion.V7_8;
+import static zipkin2.elasticsearch.ElasticsearchVersion.V9_0;
+
+import zipkin2.internal.Nullable;
+
+final class ElasticsearchSpecificTemplates extends VersionSpecificTemplates<ElasticsearchVersion> {
```

Comment:
if there is new interesting logic here, can you add to the existing unit test?

Reply:
Sure, `ElasticsearchSpecificTemplates` seems to be covered but `OpensearchSpecificTemplates`, I will add tests shortly